manifests/params.pp

```diff
@@ -44,7 +44,11 @@
       $service_name_v6 = undef
       case $::operatingsystem {
         'Debian': {
-          if versioncmp($::operatingsystemrelease, '8.0') >= 0 {
+          if $facts['os']['release']['major'] == 'buster/sid' {
```

Comment:
hello @pmelse , 
This line is causing problems. Due to that unit tests are failing. 
  $facts['os']['release']['major']  -> Evaluation Error: Operator '[]' is not applicable to an Undef Value. 
This can be easily fixed by replacing $facts['os']['release']['major'] with $::operatingsystemrelease or if you need majore release, then $:::operatingsystemmajrelease. With this fix, the tests are passing

Here is a [tutorial](https://rspec-puppet.com/tutorial/?_ga=2.215466775.1044322928.1562569909-870536227.1541596412) that may help you to write unit tests.
Please let me know if you need any help! :)